third_party/move/move-binary-format/src/file_format.rs

```diff
@@ -3444,6 +3444,34 @@ pub fn basic_test_module() -> CompiledModule {
     m
 }
 
+/// Creates an empty compiled module with specified dependencies and friends. All
+/// modules (including itself) are assumed to be stored at 0x0.
+pub fn empty_module_with_dependencies_and_friends<'a>(
```

Comment:
this file is so huge, with all the built in test infrastructure in the same file 😢 nothing to do now but should move together with tests at some point.

Reply:
Agree, an no `cfg(any(test, feature = "testing"))`, plus mocks are rather ugly...